activity_hub.php

```diff
@@ -244,19 +261,36 @@ function progress_snapshot_table($show_filtered_projects, $show_filtering_links,
             continue;
         }
 
-        $desired_states = [];
+        echo "\n<tr>";
+        echo "<th class='activity-icon-header'>&nbsp;</th>";
```

Comment:
do we need the empty space for rendering?

Reply:
I tried removing it using the DOM tools in the browser and notice no difference without it (the &nbsp;)

Reply:
I don't know but we probably don't. We presumably did waaaay back when it was developed in 2009.

Reply:
No, and upon looking at the other places in the code that had that, I have no clue why I stuck it in there in the first place. Fixed